study-builder/studies/shared/family-history/family-history-top-level.conf

```diff
@@ -0,0 +1,313 @@
+{
+  include required("../../snippets/activity-general-form.conf"),
+  "formType": "GENERAL",
+  "studyGuid": ${id.study},
+  "activityCode": "FAMILY_HISTORY",
+  "versionTag": "v1",
+  // Needs to be updated
+  "displayOrder": 1,
+  "writeOnce": true,
+  "maxInstancesPerUser": 1,
+  INSTRUCTIONS_BLOCK_SELF_EN={include required("snippets/family-history-section-instructions.conf")
+    INSTRUCTIONS_P1_EN="""
+              In this survey we would like to know the <b>living status, age, and cancer history</b> of people in your biological, or blood-related, family. We recognize that there are many different types of families, so please skip sections that do not apply to your family tree.""",
+    SHOWN_EXPR="""!user.studies[""""${id.study}""""].isGovernedParticipant()"""
```

Comment:
N.B.: Here is where we introduce slightly different language depending on whether we are dealing with a governed participant or not.